src/map/spell.cpp

```diff
@@ -698,6 +698,10 @@ namespace spell
                 // ensure trust level is appropriate+
                 if (PCaster->objtype == TYPE_TRUST && PCaster->GetMLevel() < JobMLVL && PCaster->GetSLevel() < JobSLVL)
                 {
+                    if (spell->getID() == SpellID::Stun && !PCaster->name.compare("fablinix") && PCaster->GetMLevel() >= spell->getJob(JOB_BLM))
+                    {
+                        return true;
+                    }
```

Comment:
What's happening here? We typically don't make special exceptions in the C++ code for certain spells, or entities.

If spell is stun, and anyone but fablinix casts stun, and the caster's main job level is higher than their BLM level?